lib/trace.js

```diff
@@ -23,8 +23,8 @@
  * @constructor
  */
 function Trace(projectId, traceId) {
-  this.projectId = projectId;
-  this.traceId = traceId;
+  this.project_id = projectId;
+  this.trace_id = traceId;
```

Comment:
We can, I'd like to land a functionality CL as this is already getting really big and then independently land a refactor that removes our structural classes.